### PR TITLE
add calculate game result funcs

### PR DESCRIPTION
**Pull request by talko611**

### CODE REVIEW

File: gameLoop.c [hunk 1]
```diff
@@ -20,7 +20,7 @@ void lottoLoop(int userChoice) {
 
 void startNewSimulation() {
     ParticipantList participantsList;
-    int numOfParticipants, i, winningTicket[TICKET_LEN], hitsList[TICKET_LEN];
+    int numOfParticipants, i, winningTicket[TICKET_LEN], hitsList[TICKET_LEN + 1];
```
Comment (adiveiszman) [inline gameLoop.c:23]: Rename hitList to hitArr

File: gameLoop.c [hunk 2]
```diff
@@ -253,3 +255,56 @@ void scanParticipantName(char **participantNm) {
...
+    while (ticket != NULL) {
+        setNumOfHits(ticket, winningTicket, hitArray);
+        totalNumOfHits += ticket->hits;
+        counter++;
+        ticket = ticket->next;
+    }
+    participant->avgHits = totalNumOfHits / counter;
+}
+
+void setNumOfHits(TicketNode * ticket, const int* winningTicket, int* hitArray){
```
Comment (adiveiszman) [inline gameLoop.c:281]: why is winningTicket const in this function only?

Comment (talko611) [reply gameLoop.c:281]: i will change it to all functions

File: gameLoop.c (hunk 1 above)
Comment (talko611) [reply gameLoop.c:23]: ok

**Pull request closed without merging**